src/snowflake/cli/_plugins/spcs/compute_pool/manager.py

```diff
@@ -39,7 +44,17 @@ def create(
         auto_suspend_secs: int,
         comment: Optional[str],
         if_not_exists: bool,
+        replace: bool,
     ) -> SnowflakeCursor:
+
+        if replace:
+            object_manager = ObjectManager()
+            object_type = str(ObjectType.COMPUTE_POOL)
```

Comment:
```suggestion
object_type = ObjectType.COMPUTE_POOL.sf_name
```

It's the same, but seems more informative